### DISCUSSION TITLE
Duda HTML

### DISCUSSION THREAD

**Discussion by ArmandoRiveraT** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#260] [Q&A]
Hola compañeros, me encontraba haciendo una mis paginas html y todo bien, el problema es que al querer copiar y pegar el código para hacer una nueva pagina de html no me aparece igual como la primer pagina, me podrían ayudar? adjunto imágenes 
Código de mi css:
.menufondo{
    background-color: 630A10;
    margin:0; padding:0;
    height:70px;
   
}

.imagenlogo{
    position:relative;
    bottom: 13px;
z-index: 1;
}

.fondo{
    background-color: FCF0C8;
    margin:0%; padding:0%;
}

.alinearmenu{
position:relative;
    left: 1000px;
    bottom: 35px;
    z-index: 2;
}

.boton {
    background-color: rgb(145,145,145); 
    color: white!important;
    border: 14px;
}
    
.boton:hover {
        background-color: #31495f; 
    }

.slider{
    width:1000px; 
    margin:auto;
    overflow: hidden;
   
}

.slider ul {
    display:flex;
    padding:0;
    width:1000px;
   

    animation: cambio 20s infinite alternate linear;
    

    }

.slider li {
    width:1000px;
    height: 450px;
    list-style:none;
    
}

.slider img{
    width:1000px;
    height:450px;
    
}

@keyframes cambio{
    0%{margin-left: 0%;}
    33%{margin-left: 0%;}

    34%{margin-left: -100%;}
    67%{margin-left: -100%;}
    
    68%{margin-left: -200%;}
    100%{margin-left: -200%;}
}

.divslid{
    background-color: white;
}

/*sabores pagina*/ /*sabores pagina*/ /*sabores pagina*/
.titulo{
    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
    font-size:60px;
    text-align: center;
}

.textoo{
    text-align: center;
}

.fondoprecio{
    background-color: orange;
}

.subfondo{
    background-color: 630A10;
    height: 100px;

}
.direccion{
    position:relative;
    left:390px;
    top:10px;
}
.siguenos{
    position: relative;
    left:950px;
    bottom: 70px;
}

.textodesubmenu{
    color: aliceblue;
}

.icon{
    width: 30px;
    height:30px;
}
.tel{
    position: relative;
    left:35px;
    bottom: 140px;
}

.horario{
    position: relative;
    left:700px;
    bottom: 200px;

}
<img width="960" alt="Captura de pantalla 2022-03-05 164459" src="https://user-images.githubusercontent.com/99206229/156902227-faf6c61a-b298-4e8b-90f9-f36b8a5f6478.png">
<img width="960" alt="Captura de pantalla 2022-03-05 164521" src="https://user-images.githubusercontent.com/99206229/156902230-03a31706-674b-4926-8aea-f30d07b94985.png">
<img width="960" alt="CAPTURA1" src="https://user-images.githubusercontent.com/99206229/156902231-d0fa2478-dfd4-42d8-b3a0-5207cfdd3862.png">

**Comment by YomairaBarrera** [comment]
No tengo tanta experiencia y la verdad no se si afecte en algo, pero a algunos de tus colores en hexadecimal le falta el #

**Reply by ArmandoRiveraT** [reply]
AJAJAJ si era eso gracias. Lo raro es que para la primera pagina no puse el # y como quiera me mostraba el color (como se ve en la primer imagen), pero ya en la segunda pagina web no me arrojaba los colores. Bueno, muchas gracias :)

**Reply by YomairaBarrera** [reply]
:smile: Yeiii que bueno que funciono, de nada

**Comment by Ingrzs** [comment]
1- el archivo lo tienes guardado en la misma carpeta que los otros?
2-tal ves te falte en el head el rel de tu archivo style.

**Reply by ArmandoRiveraT** [reply]
Hola, si el archivo lo tengo en la misma carpetas, lo que faltaba era poner el # en el color, raro por que en la primer maqueta que hice si me mostraba los colores bien y sin tener el #, pero ya en la segunda maqueta ya no me mostraba nada jaja. Bueno, gracias por comentarme 👍 :)

**Comment by RodolfoMorquecho** [comment]
Disculpa, esto esta muy fuera de contexto y espero me puedas ayudar. Estoy terminando mi pagina y utilice imagenes de google para los pasteles pero el MC menciono varias veces que debiamos usar imagenes sin derechos de autor, tu buscaste en algun sitio especial? o en este caso no importa?

**Comment by hackit92** [comment]
A lo mejor no estoy viendo completo pero no veo que en hola.html este cerrado el </body> </html>, sería más fácil ver también el código fuente, pero podría deberse a eso que no está renderizando la página

**Reply by hackit92** [reply]
de igual forma veo que acomodas los textos en tu CSS con coordenadas, te recomiendo utilizar CSS Grid para ello o flexbox, con lo que te sientas mas cómodo
